<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAAlignmentTool</class>
 <widget class="QWidget" name="CAAlignmentTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bessel beam alignement tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCustomPlot" name="focusLongitudinalProfile" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="peakPositionProfile" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="transversProfile" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
